app/assets/javascripts/Components/Modals/send_lti_grades_modal.jsx

```diff
@@ -39,7 +39,7 @@ class LtiGradeModal extends React.Component {
     );
     const deploymentsToUpdate = checked.map(deployment => deployment[0]);
     $.post({
-      url: Routes.lti_deployment_create_lti_grades_path(),
+      url: Routes.create_lti_grades_lti_deployment_index_path(),
```

Comment:
This looked quite strange to me, since "collection" routes normally do not need an `index` suffix.

I think you aren't quite following the Rails naming conventions for resourceful routing:

1. In the routes.rb file, the "resources" should be pluralized (`lti_deployments`)
2. The controller filename and class name should be pluralized (`lti_deployments_controller.rb`, `LtiDeploymentsController`

I think after you make this change you should be able to use the path helper `create_lti_grades_lti_deployments_path`, which follows a more standard naming style (as with other uses of "`_index_path`" below).